snuba/admin/views.py

```diff
@@ -520,41 +553,37 @@ def clickhouse_trace_query() -> Response:
             400,
         )
     except ClickhouseError as err:
+        logger.error(err, exc_info=True)
```

Comment:
We should probably add logger.error() in all except blocks in this file. I found it quite useful in debugging multiple failures.